assets/js/item_details.js

```diff
@@ -8,6 +8,10 @@ const uploadedImages = []
 
 $(".tagsInputBoxContainer").hide();
 
+if (!$("#locationSelect option:selected").length) {
```

Comment:
Something weird is going on here.  You are getting the length of a selector and then negating that?

Reply:
0 = false, if it's zero the boolean is negated

Reply:
So I don't have strong feelings on this, but I would make the == 0. more explicit and remove the !

Reply:
fine, i'll change it. both you and justin have suggested the same thing.